doc/api/next_api_changes/removals.rst

```diff
@@ -73,6 +73,14 @@ Classes and methods
 - ``scale.NaturalLogTransform`` (use ``scale.LogTransform`` instead)
 - ``scale.InvertedNaturalLogTransform`` (use ``scale.InvertedLogTransform`` instead)
 
+- ``spines.Spine.is_frame_like()`` (no replacement)
```

Comment:
What should a user do if they were using `is_frame_like` in their code? It seems to me that we should at least be justifying why there is no replacement, if not offering an alternative.

Reply:
This was considered not to be of practical use, see #13393.

We have the deprecation so that people can speak up if the removal affects them. Since nobody did, this is a rare or even unused feature. I‘d just proceed and think of an alternative (which may be copying the existing code) if the problem comes up.